Documentation/source/guide/tutorials-imagesequence.rst

```diff
@@ -113,25 +120,13 @@ It is can also specify the framerate and the codec, here is an example for frame
 
 ``ffmpeg -framerate 30 -i input%04d.png -c:v libx264 -r 30 -pix_fmt yuv420p out.mp4``
 
-Using an NLE
-============
+Using NLE
+==========
 
-Kdenlive
-~~~~~~~~
+Kdenlive, Shotcut, Da Vinci Resolve, Adobe Premiere
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Import the image "as sequence" in the timeline (or drag'n'drop the folder) and render in your preferred video format. 
 
-Shotcut
-~~~~~~~
-
-Da Vinci Resolve
-~~~~~~~~~~~~~~~~
-
```

Comment:
- In Resolve, select the image sequence(s).
- Go to File => Media Management.
- Select Clips.
- **Select "Transcode" mode.**
- Select Media Destination.
- Select _the desired_ Video Format.

Reply:
Ignore this suggestion if you prefer the general method for NLE.